File: 1428/1428.c
int leftmostColumnWithAtLeastAOne(struct TreeNode* root) {
    if (!root) return 0;
    int l = leftmostColumnWithAtLeastAOne(root->left);
    int r = leftmostColumnWithAtLeastAOne(root->right);
    return 1 + (l > r ? l : r);
}

File: 1428/1428.cs
public class Solution {
    public int LeftmostColumnWithAtLeastAOne(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(LeftmostColumnWithAtLeastAOne(root.left), LeftmostColumnWithAtLeastAOne(root.right));
    }
}